package.json

```
@@ -223,6 +223,7 @@
 		"fixtures:generate": "cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit",
 		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
 		"format-js": "wp-scripts format-js",
+		"format-php": "wp-env run composer run-script format",
```

I'm on Ubuntu and don't use Docker...
This is what I get when I run `npm run format-php`:
```
ERROR: Couldn't connect to Docker daemon at http+docker://localhost - is it running?
```

Changing the line to this fixes the issue on my end:
```suggestion
		"format-php": "composer run-script format",
```
A prerequisite is of course that I have first ran `composer install`, but I think that is a reasonable assumption.
Why is `wp-env` a requirement for this?

---

This new command follows the same model as the existing `lint-php` and `test-unit-php`. Is there any reason this command should be different?

---

I've actually never used the `lint-php` and `test-unit-php` commands... Usually I just run `vendor/bin/phpcs` from a terminal so tbh I didn't even know they existed.
With that new piece of information I think we're OK to leave it as is... Those that use wp-env can use these commands. The rest of us that don't use wp-env are used to running phpcs/phpcbf manually anyway so nothing changes :+1: